### ISSUE TITLE
Request: Support for import .osp files into the project

### ISSUE THREAD

**Issue by peanutbutterandcrackers** [OpenShot/openshot-qt#1230]
So, turns out Kdenlive has this feature where one can import a ready-made kdenlive project into another kdenlive project (and it imports like a video). A YouTuber used this feature to insert the project he was using to teach at the end of his own screencast tutorial video. I think OpenShot should implement this feature too.

Also, if it can be done: if OpenShot could somehow also import kdenlive project, and if, while rendering, OpenShot could ask kdenlive to render that section of the video which is a kdenlive project, we the users would have all the keyframe-prowess of OpenShot and all the other stuffs that kdenlive has got going for itself. I know it might sound stupid, but I think that'd be an amazing feature. Only, make kdenlive a must-have-installed package if that feature is to be used.

OpenShot Version: 2.4.1-dev1

**Comment by peanutbutterandcrackers** [comment]
@DylanC - Apparently, this one's kinda' a duplicate of #729, only a little more ambitious proposal. :D

**Comment by DylanC** [comment]
@peanutbutterandcrackers  - Sounds cool to me!

**Comment by peanutbutterandcrackers** [comment]
@DylanC - Hehe. Thanks! :)

**Comment by MarkusPiotrowski** [comment]
Any plans to implement this? Due to the Corona crisis I'm making lab videos for our students. After doing two videos I realized that I want to re-arrange some scenes (e.g. move them from video 1 to video 2 with the existing transitions, overlays etc). Naively I thought I can simply copy & paste them between projects. Then I tried importing... 
First rendering them to video is quite painful. Copy & paste support between projects or appending another .osp file would be great.
